modules/end-user-guide/partials/proc_building-an-image-for-an-ide-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,160 @@
++
+.Building the Docker image with IntelliJ IDEA Community 2020.3.3
+====
+----
+./projector.sh build --tag che-idea:2020.3.3 \
+  --url https://download-cdn.jetbrains.com/idea/ideaIC-2020.3.3.tar.gz
+----
+====
++
+.Building the Docker image with PyCharm Community 2020.3.5
+====
+----
+$ ./projector.sh build --tag che-pycharm:2020.3.5 \
+  --url https://download.jetbrains.com/python/pycharm-community-2020.3.5.tar.gz
+----
+====
++
+.Building the Docker image with WebStorm 2020.3.3
+====
+----
+$ ./projector.sh build --tag che-webstorm:2020.3.3 \
+  --url https://download.jetbrains.com/webstorm/WebStorm-2020.3.3.tar.gz
+----
+====
++
+.Building the Docker image with IntelliJ IDEA Ultimate 2020.2.2
+====
+----
+$ ./projector.sh build --tag che-idea-ultimate:2020.2.2 \
+  --url https://download.jetbrains.com/idea/ideaIU-2020.2.2.tar.gz
+----
+====
+
+. To test the image, run it locally, and navigate to `++http://localhost:8887++` to access the IDE.
++
+[subs="+quotes,macros,attributes"]
+----
+$ ./projector.sh run __<tag>__
+----
++
+.Testing the Docker image with IntelliJ IDEA Community 2020.3.3
+====
+----
+$ ./projector.sh run che-idea:2020.3.3
+----
+====
++
+.Testing the Docker image with PyCharm 2020.3.5
+====
+----
+$ ./projector.sh run che-pycharm:2020.3.5
+----
+====
++
+.Testing the Docker image with WebStorm 2020.3.3
+====
+----
+$ ./projector.sh run che-webstorm:2020.3.3
+----
+====
++
+.Testing the Docker image with IntelliJ IDEA Ultimate 2020.2.2
+====
+----
+$ ./projector.sh run che-idea-ultimate:2020.2.2
+----
+====
+
+. Publish the image to a registry accessible by {prod-short}, and remember the location: __<image>__.
+
+. Create a `+meta.yaml+` file with the following content:
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+apiVersion: v2
+publisher: __<publisher>__ <1>
+name: intellij-ide
+version: latest
+type: Che Editor
+displayName: IntelliJ Platform IDE
+title: IDE based on the IntelliJ Platform as Editor for Eclipse Che
+description: IDE based on the IntelliJ Platform running using Projector
+icon: https://www.jetbrains.com/apple-touch-icon.png
+category: Editor
+repository: https://github.com/che-incubator/jetbrains-editor-images
+firstPublicationDate: "2021-04-10"
+spec:
+  endpoints:
+   -  name: "intellij"
+      public: true
+      targetPort: 8887
+      attributes:
+        protocol: http
+        type: ide
+        path: /projector-client/index.html?backgroundColor=434343&wss
+  containers:
+   - name: intellij-ide
+     image: "__<image>__" <2>
+     mountSources: true
+     volumes:
+         - mountPath: "/home/projector-user"
+           name: projector-user
+     ports:
+         - exposedPort: 8887
+     memoryLimit: "4096M"
+----
+<1> __<publisher>__: Your publisher name.
+<2> __<image>__: Location of the editor image in a registry accessible by {prod-short}.
+
+. Publish the `meta.yaml` file to an HTTPS resource accessible by {prod-short}: `__<URL_to_meta.yaml>__`.
```

Comment:
Looks like `__<URL_to_meta.yaml>__` is an orphan here.

Reply:
```suggestion
. Publish the `meta.yaml` file to an HTTPS resource accessible by {prod-short} and copy the full path to the file for use as `__<URL_to_meta.yaml>__` when configuring a workspace devfile for this IDE image.
```

Reply:
`full path` is misleading.

Reply:
No reason to instruct the user to 'copy'.